nemo_aligner/utils/distributed.py

```diff
@@ -317,6 +324,10 @@ def get_synced(self, *args, **kwargs):
         # sync the time
         return self.sync_time([output]).item()
 
+    def stop_and_get_time(self, name=""):
```

Comment:
Doesn't seem to be used anywhere?

Reply:
ah sorry, artifact of the way I picked apart the TRT-LLM changes and my critic change.

15fc308cb70a26758cb9ebaf9bc01a6c65f2dadd